src/app/components/surveillance/reporting/relevant-listings.component.js

```diff
@@ -0,0 +1,52 @@
+export const SurveillanceReportRelevantListingsComponent = {
+    templateUrl: 'chpl.components/surveillance/reporting/relevant-listings.html',
+    bindings: {
+        listings: '<',
+        onSave: '&?',
+    },
+    controller: class SurveillanceReportRelevantListingComponent {
+        constructor ($log) {
+            'ngInject'
+            this.$log = $log;
+            this.safeListings = [];
+        }
+
+        $onChanges (changes) {
+            if (changes.listings) {
+                this.listings = angular.copy(changes.listings.currentValue);
+                if (Array.isArray(this.listings)) {
+                    this.listings.forEach(listing => {
+                        listing.formattedCertificationDate = new Date(listing.certificationDate);
+                        listing.lastModifiedDate = parseInt(listing.lastModifiedDate, 10);
+                        listing.formattedLastModifiedDate = new Date(listing.lastModifiedDate);
+                    })
+                }
+                this.safeListings = angular.copy(this.listings);
```

Comment:
I've generally used `this.backup` as `{}` to save backup copies of stuff. Would like to keep consistent on that if we can, so line 11 above would be `this.backup = {};`
```suggestion
                this.backup.listings = angular.copy(this.listings);
```